app/templates/application.hbs

```diff
@@ -11,4 +11,17 @@
 </nav>
 <div id="application-container">
   {{outlet}}
+<nav id="top-menu" {{bind-attr class='isExpanded:lowered:raised'}}>
```

Comment:
should be changed to `class="{{if isExapnded 'lowered' 'raised'}}"` to remove the depreciation warning.